[Failing Test] integration-master (ci-kubernetes-integration-master)

<!-- Please only use this template for submitting reports about flaky tests or jobs (pass or fail with no underlying change in code) in Kubernetes CI -->

**Which jobs are flaking**:

```
ci-kubernetes-integration-master
```

**Which test(s) are flaking**:

```
k8s.io/kubernetes/test/integration/client.TestDynamicClient
```

**Testgrid link**:

https://testgrid.k8s.io/sig-release-master-blocking#integration-master

**Reason for failure**:

```shell
--- FAIL: TestDynamicClient (6.68s)
    testserver.go:181: runtime-config=map[api/all:true]
    testserver.go:182: Starting kube-apiserver on port 37255...
    testserver.go:198: Waiting for /healthz to be ok...
    dynamic_client_test.go:88: unexpected pod in list.  ....
```

**Anything else we need to know**:
- https://storage.googleapis.com/k8s-gubernator/triage/index.html?pr=1&text=dynamic_client_test.go%3A88%3A%20unexpected%20pod%20in%20list.&job=ci-kubernetes-integration-master&test=k8s.io%2Fkubernetes%2Ftest%2Fintegration%2Fclient.TestDynamicClient
- https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-integration-master/1216620112996470785

/cc @kubernetes/ci-signal 
/sig api-machinery
/priority critical-urgent

--- a/staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager.go
+++ b/staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager.go
@@ -35,7 +35,7 @@ const DefaultMaxUpdateManagers int = 10
 
 // DefaultTrackOnCreateProbability defines the default probability that the field management of an object
 // starts being tracked from the object's creation, instead of from the first time the object is applied to.
-const DefaultTrackOnCreateProbability float32 = 0.5
+const DefaultTrackOnCreateProbability float32 = 0.0
 
 // Managed groups a fieldpath.ManagedFields together with the timestamps associated with each operation.
 type Managed interface {

